### PR TITLE
fix(editor): Enable moving resources only if team projects are available by the license

### PR DESCRIPTION
**Pull request by cstuncsik**
## Summary

The current setting of the Move action is broken. We can't rely on the plan name because it is the same on the cloud, where we should enable moving resources even when the plan name shows `Community` if the users have a quota for team projects.

## Related Linear tickets, Github issues, and Community forum posts

[PAY-1798](https://linear.app/n8n/issue/PAY-1798/bug-cannot-rely-on-planname-to-check-for-permission-to-move-workflows)

## Review / Merge checklist

- [x] PR title and summary are descriptive
- [x] Tests included
- [x] PR Labeled with `release/backport`

### CODE REVIEW

File: packages/editor-ui/src/stores/projects.store.ts [hunk 1]
```diff
@@ -49,19 +49,19 @@ export const useProjectsStore = defineStore('projects', () => {
 	);
 	const teamProjects = computed(() => projects.value.filter((p) => p.type === ProjectTypes.Team));
 	const teamProjectsLimit = computed(() => settingsStore.settings.enterprise.projects.team.limit);
-	const teamProjectsAvailable = computed<boolean>(
+	const isTeamProjectsAvailable = computed<boolean>(
```
Comment (tomi) [inline packages/editor-ui/src/stores/projects.store.ts:52]: 👍 much better with the prefix. Tho to be grammatically correct, we should name this `areTeamProjectsAvailable`, as it's plural project**s**

**Review by tomi — DISMISSED** [review]
Nice 👏 Just one small comment about grammar

**Review by tomi — APPROVED** [review]
Thanks for addressing 💟

**Comment by janober** [comment]
Got released with [`n8n@1.53.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.53.1)

**Pull request merged**